utils/cloze_utils.py

```diff
@@ -27,6 +27,7 @@ def cloze_input_parser(cloze: str) -> str:
     if you use weird formatting that mess with LLMs"""
     assert iscloze(cloze), f"Invalid cloze: {cloze}"
 
+    # TODO: What is this?
     cloze = cloze.replace("\xa0", " ")
```

Comment:
I don't recall exactly but I encountered this very frequently. I think it's something anki used in lieu of `&nbsp;`